regexes/bots.yml

```diff
@@ -164,7 +164,7 @@
     name: '2.0Promotion GbR'
     url: 'http://www.backlinktest.com'
 
-- regex: 'baiduspider(-image)?|baidu Transcoder|baidu.*spider'
+- regex: '[Bb]aidu.*spider|baidu Transcoder'
```

Comment:
`baidu.*spider|baidu Transcoder`

all regular expressions are executed with the flag `i`

https://github.com/matomo-org/device-detector/blob/975c63703f76e0831332b8310b40b0c9113dd003/Parser/AbstractParser.php#L248-L253

Reply:
Ah okay, I wasn't expecting it else all other regexes have correct capitalisation. I'll keep the capital `B` then, as this is really correct for all known Baiduspider user agents.

I also recognised that `baidu.*spider` matched our two examples already due to the URL `http://www.baidu.com/search/spider.html`, even with lower case `b`. So it remains a little regex simplification.